web/api/backend.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"time"
+	jwtmiddleware "github.com/auth0/go-jwt-middleware"
+	"github.com/dgrijalva/jwt-go"
+)
+
+// Credentials object read body from the request body
+type Credentials struct {
+	RequestType string
+	Args        map[string]interface{}
+	UseCache    string
+	Body        map[string]interface{}
+}
+
+// Claims object that will be encoded to a JWT.
+// We add jwt.StandardClaims as an embedded type, to provide fields like expiry time
+type Claims struct {
+	Body map[string]interface{}
+	jwt.StandardClaims
+}
+
+var jwtKey = []byte("my_secret_key")
+var creds Credentials
+
+// TokenHandler to create the Token
+func TokenHandler(w http.ResponseWriter, r *http.Request) {
+	// Get the JSON body and decode into credentials
+	err := json.NewDecoder(r.Body).Decode(&creds)
+	if err != nil {
+		// If the structure of the body is wrong, return an HTTP error
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+	if len(creds.Body) == 0 {
+		w.WriteHeader(http.StatusBadRequest)
+		io.WriteString(w, `{"error":"cannot make token without credentials"}`)
+		return
+	}
+
+	// Declare the expiration time of the token
+	// here, we have kept it as 5 minutes
+	expirationTime := time.Now().Add(5 * time.Minute)
```

Comment:
Don't forget to extend the expiration time!

Reply:
Extended time to a day